source/website/Slides/0.js

```diff
@@ -3,7 +3,7 @@ import { TitleSlide } from '../../modules'
 
 const slide = () => (
   <TitleSlide>
-    <h1>React Presents: a slideshow framework</h1>
+    <h1 style={{color: 'white'}}>React Presents: a slideshow framework</h1>
```

Comment:
Not a fan of this particularly. Should be set by the parent slide.

Why wouldn't the default for CSS be `color: inherit` ?